src/panel.tsx

```diff
@@ -11,46 +11,47 @@ const ABSTRACT_APP_URL =
 const Iframe = styled.iframe({
   width: "100%",
   height: "100%",
-  border: "0 none"
+  border: "0 none",
 });
 
 function parseShareURL(url: string): string | void {
   if (url.match(/share.(?:go)?abstract.com\//)) {
     const parsedUrl = new URL(url);
-    const pathSegments = parsedUrl.pathname.split('/');
+    const pathSegments = parsedUrl.pathname.split("/");
     return pathSegments[pathSegments.length - 1];
   }
 
   throw new Error(
-    `The provided url (${url}) is not valid. The url must come from "https://share.abstract.com/" or "https://share.goabstract.com/".`,
+    `The provided url (${url}) is not valid. The url must come from "https://share.abstract.com/" or "https://share.goabstract.com/".`
   );
 }
 
 type AddonOptions = {
-  theme?: "light" | "dark" | "system",
-  chromeless?: boolean
+  theme?: "light" | "dark" | "system";
+  chromeless?: boolean;
 };
 
 export function Panel() {
   const { storyId } = useStorybookState();
-  const params = useParameter<(Abstract.ShareDescriptor | Abstract.ShareUrlDescriptor) & {
-    options?: AddonOptions  
-  }>(
-    PARAM_KEY
-  );
+  const params = useParameter<
+    (Abstract.ShareDescriptor | Abstract.ShareUrlDescriptor) & {
+      options?: AddonOptions;
+      slug?: string;
```

Comment:
This is great! My only suggestion is to call this `find` through and through

Reply:
I renamed "slug" to "find" in another commit ;)